.generated-sources/blocklist-api/Cargo.toml

```diff
@@ -10,6 +10,7 @@ edition = "2021"
 serde = { version = "^1.0", features = ["derive"] }
 serde_with = { version = "^3.8", default-features = false, features = ["base64", "std", "macros"] }
 serde_json = "^1.0"
+serde_repr = "^0.1"
```

Comment:
Hmmm, `make install && make build` must have done this.

Reply:
Locally I have the same, I just never bothered to commit it.

Reply:
Marking this as "resolved" for now so it doesn't block this PR but happy to open an issue on general 'git is dirty' / 'make install' 'make build,' I imagine it'll happen again.